Formula/c/cling.rb

```diff
@@ -8,7 +8,7 @@ class Cling < Formula
 
   livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
```

`stable` isn't a GitHub release asset, so we should document why we're using `GithubLatest` here.